### ISSUE TITLE
Edit - Setttings and View Tabs doesn't present any options

### ISSUE THREAD

**Issue by gookcedemir** [anujdatar/onenote-desktop#29]
First of all, thank you for improving OneNote, I actively use it in Linux, Windows and Mac.

On my Linux, the new version has bug on Edit, View and Settings Tabs. Because all three shows broken content instead of listing options (regardless of opening my notebook) which you can see the images with the following link: [OneNote Bug](https://drive.google.com/drive/folders/1Ui6s6hZT6AsiLgSYN5DkGw9BmqMqE-74?usp=sharing)

As a feature request, it would be nice if the OneNote remembers the last opened notebook and shows it directly when it is started. I don't like to go Navigate-> OneNote Home every time when I start the oneNote app. I think this feature was already supported on the old version.

**Comment by anujdatar** [comment]
> First of all, thank you for improving OneNote, I actively use it in Linux, Windows and Mac.

You're very welcome. The update was long overdue.

> On my Linux, the new version has bug on Edit, View and Settings Tabs. Because all three shows broken content instead of listing options (regardless of opening my notebook) which you can see the images with the following link: [OneNote Bug](https://drive.google.com/drive/folders/1Ui6s6hZT6AsiLgSYN5DkGw9BmqMqE-74?usp=sharing)

That is curious. Looks like some of those menu options are not being rendered correctly by Electron or your DE. I could use a bit more information to reproduce and fix the problem at my end:
1. Which version of Linux you are using? *Ubuntu/Arch/Fedora and the release.*
2. What's your desktop environment/window manager? *Gnome, KDE, XFCE, Openbox etc*
3. Are you using X11 or Wayland?
4. Which package are you using? The `.deb`, `.AppImage` or `Snap` package?
5. Also, it looks like `Navigate` and `Help` are working fine?
My testing enviroments are Ubuntu 20.04 and 21.10, and Manjaro. but I mostly use KDE and XFCE. From the images it seems like your have Ubuntu with Gnome
 
> As a feature request, it would be nice if the OneNote remembers the last opened notebook and shows it directly when it is started. I don't like to go Navigate-> OneNote Home every time when I start the oneNote app. I think this feature was already supported on the old version.

The app should load the last opened notebook when the app is relaunched. I'll check this behavior. I had changed it during testing. Let me see if I forgot to change it back.

**Comment by anujdatar** [comment]
I apologize for the inconvenience. But, while I try to test this, would you mind trying out the following steps?

___
## Method 1:
This will only clear local app settings and reset to default. Your login and app history and everything OneNote related will stay intact. This app does not directly make any changes to OneNote.
A. If you installed the app using the `.deb` package:
1. Close the app
2. Go to `/home/<user>/.config/onenote-desktop` or `/home/<user>/.config/OneNote`
3. Delete the `config.json` file
4. Restart the app.
OR
B. If you are using `.snap`, i.e. you used `sudo snap install onenote-desktop` or used the Ubuntu App Store to install the app
1. Close the app
2. Go to `/home/<user>/snap/onenote-desktop/current/.config/OneNote/`
3. Delete the `config.json` file
4. Restart the app.

___
## Method 2:
If you don't mind losing you instance login. You will NOT lose any OneNote data. But you will just need to login again, and app settings will be reset to defaults. Not OneNote settings, just the Linux wrapper app.
Instead of deleting the `config.json` files from method 1, delete the entire config folder. I would recommend this step, but only if you have your login information and can easily login again.
1. I.e. if `.deb` installation, delete the folder `/home/<user>/.config/onenote-desktop` or `/home/<user>/.config/OneNote`
2. if `Snap` installation, delete the folder `/home/<user>/snap/onenote-desktop/current/.config/OneNote/`

___
## Method 3:
You can achieve either or these results if you use the new `Reset App Defaults` (soft reset, Method 1) or `Reset App - everything` (hard reset, Method 2) options. You can find them in the `Help` section of the app menu.

I would really appreciate your feedback if you try any of these. Thanks

**Comment by anujdatar** [comment]
Hi, I just ran a few checks and all the menu items seem to render just fine for me. It could be that the Electron version update is causing some graphics glitches. I recommend manually resetting the app once. **Method 2** mentioned above. It could be that the app settings and caches are not migrating correctly after the upgrade.


> As a feature request, it would be nice if the OneNote remembers the last opened notebook and shows it directly when it is started. I don't like to go Navigate-> OneNote Home every time when I start the oneNote app. I think this feature was already supported on the old version.

About this, the app still remembers your last the last opened notebook and that should launch when the app is started again.


For more context, I just tested the app on Ubuntu 20.04 and 2.10, with XFCE, LXQT, Gnome and KDE desktop environments. Everything worked as intended. I was unable to reproduce the errors you are experiencing. A bit more information would be helpful.

Thanks, Anuj
